test/org/sagebionetworks/bridge/dynamodb/DynamoUserConsentDaoTest.java

```diff
@@ -28,14 +28,21 @@
     @Before
     public void before() {
         DynamoInitializer.init(DynamoUserConsent2.class);
-        DynamoTestUtil.clearTable(DynamoUserConsent2.class);
+        DynamoInitializer.init(DynamoUserConsent3.class);
```

Comment:
DynamoInitializer.init() is incredibly slow, even for tables that already exist. Can we move these to a @BeforeClass so we only need to run these once?